### PR TITLE
Added Framework Badge CSS Override

### PR DESCRIPTION
**Pull request by spider-sauce**
### Description

- Added `fw-override.css`
- Added sytles for each STT framework (we can add more as we progress)

### Motivation and Context

- Easer to manage than react component
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/saucectl.md [hunk 1]
```diff
@@ -113,7 +106,7 @@ saucectl run --suite <suite_name>
 
 ### `test-env`
 
-<p><small>supported frameworks: <Highlight color="#25c2a0">cypress</Highlight></small></p>
+<p xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"><small>supported frameworks: <span class="highlight cypress">cypress</span></small></p>
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/saucectl.md:109]: What is this reference inside the paragraph tag? Why is it here twice?

Comment (spider-sauce) [reply docs/testrunner-toolkit/saucectl.md:109]: I have no idea what this is: `xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"`, we can remove it.

Comment (spider-sauce) [inline docs/testrunner-toolkit/saucectl.md:109]:
```suggestion
<p><small>supported frameworks: <span class="highlight cypress">cypress</span></small></p>
```

**Review by sweeneyskirt-sl — APPROVED** [review]
LGTM

**Pull request merged**